hopsworks-api/src/main/java/io/hops/hopsworks/api/featurestore/featuregroup/FeaturegroupService.java

```diff
@@ -392,7 +393,9 @@ public Response updateFeaturegroup(
       @QueryParam("updateStats") Boolean updateStats, @ApiParam(value = "enableOnline", example = "true",
     defaultValue = "false") @QueryParam("enableOnline") Boolean enableOnline,
     @ApiParam(value = "disableOnline", example = "true", defaultValue = "false") @QueryParam("disableOnline")
-      Boolean disableOnline, FeaturegroupDTO featuregroupDTO)
+      Boolean disableOnline, @ApiParam(value = "updateStatsSettings", example = "true", defaultValue =
+    "false") @DefaultValue("false") @QueryParam("updateStatsSettings") Boolean updateStatsSettings, FeaturegroupDTO
```

Comment:
I don't think you need the `@DefaultValue` here, the `defaultValue` field in the `@ApiParam` annotation should be enough.